Nimble/Matchers/RaisesException.swift

```diff
@@ -10,14 +10,14 @@ import Foundation
 /// nil arguments indicates that the matcher should not attempt to match against
 /// that parameter.
 public func raiseException(
-    named: String? = nil,
+    named named: String? = nil,
```

Comment:
I believe this is meant to read "*expect* this closure *to* *raise* an *exception* *named*...", and thus the name of this parameter.

In any case, that's the current API. Changing it should be part of a separate pull request.